src/fsharp/CompileOptions.fs

```diff
@@ -821,19 +822,17 @@ let mlCompatibilityFlag (tcConfigB: TcConfigBuilder) =
 
 let languageFlags tcConfigB =
     [
-        CompilerOption
-            ("checked", tagNone,
-             OptionSwitch (fun switch -> tcConfigB.checkOverflow <- (switch = OptionSwitch.On)), None,
-             Some (FSComp.SR.optsChecked()))
-        
-        CompilerOption
-            ("define", tagString,
-             OptionString (defineSymbol tcConfigB), None,
-             Some (FSComp.SR.optsDefine()))
-        
+        // -langversion:?                Display the allowed values for language version
+        // -langversion:<string>         Specify language version such as
+        //                               `default` (latest major version), or
+        //                               `latest` (latest version, including minor versions),
+        //                               `preview` (features for preview)
+        //                               or specific versions like `6` or `7.1`
```

Comment:
These don't exist (and may not exist for a long, long time).

Change comment to: `or specific versions like '4.7'`

Reply:
Sure.